wiki/Tournaments/RDOC/2020/en.md

```diff
@@ -0,0 +1,131 @@
+| Streamer | ![][flag_DK] [Gladand](https://osu.ppy.sh/users/8996506), ![][flag_DK] [MadX](https://osu.ppy.sh/users/3295048), ![][flag_DK] [My Aim Zogs](https://osu.ppy.sh/users/3722715), ![][flag_DK] [Tenaxzity](https://osu.ppy.sh/users/7395529) |
+| Commentator | ![][flag_DK] [ADanishWaffle](https://osu.ppy.sh/users/8117977), ![][flag_MX] [Baskest](https://osu.ppy.sh/users/3963408), ![][flag_DK] [Haikuna](https://osu.ppy.sh/users/3786974), ![][flag_DK] [Peperonni](https://osu.ppy.sh/users/7392020), ![][flag_DK] [PolleSving](https://osu.ppy.sh/users/13218204), ![][flag_DK] [Power Supply](https://osu.ppy.sh/users/7091513), ![][flag_DK] [Tenaxzity](https://osu.ppy.sh/users/7395529), ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048), ![][flag_DK] [Zakura](https://osu.ppy.sh/users/3053599), ![][flag_DK] [Zeezus](https://osu.ppy.sh/users/9125328) |
+| Designer | ![][flag_FI] [Micro](https://osu.ppy.sh/users/4801080), ![][flag_BR] [Hana Fubuki](https://osu.ppy.sh/users/4761039) |
+| Wiki editor | ![][flag_DK] [Pennek](https://osu.ppy.sh/users/3723742) |
+| Donator | ![][flag_DK] [MadX](https://osu.ppy.sh/users/3295048), ![][flag_DK] [Pennek](https://osu.ppy.sh/users/3723742) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1097554)
+- [RDOC Discord server](https://discord.gg/JhBkmzE)
+- [Livestream](https://www.twitch.tv/royaldanishosucup)
+- [Challonge brackets](https://challonge.com/rdoc2020)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/19NJkGHDmvXprqqO1wr9g2mfkkepGYmW5nXSsd6KHMk4/edit?usp=sharing)**
+
+## Ruleset
+
+### General rules
+
+1. Only users in the Kingdom of Denmark can enter this tournament. This is determined by the profile-flag on a player's userpage (e.g. a player with either of these flags is eligible to participate: ![][flag_DK], ![][flag_FO], ![][flag_GL]) (note that it is **NOT** mandatory to understand danish)
+2. This tournament uses a qualifier into a double-elimination bracket and single-player format. There are no teams here.
+3. To participate you **must** join the Discord for the duration of the tournament.
+4. Every stage will last one weekend.
+5. If a player is found to be abusive or rude to other players in the tournament, they may be removed from the tournament at the organiser's discretion. This will never be done without talking to the parties involved. If conduct is bad enough, players may be blacklisted from future tournaments and reported to osu! moderation staff. Trashtalk is fine—harrasment is not.
+6. Any other rules changes or unexpected occurrences will be announced on the Discord server.
+7. Since this is a badged tournament you will only be able to participate after having passed a screening-test for any current or previous suspicious activity from osu!staff. This means that if you've been banned within the last year you will be unable to participate in the tournament.
```

Comment:
```suggestion
7. Since this is a badged tournament, you will only be able to participate after having passed a screening-test for any current or previous suspicious activity from osu!staff. This means that if you've been banned within the last year you will be unable to participate in the tournament.
```